bites.py

```diff
@@ -168,3 +168,36 @@ def initialise_anthill(ants):
   """
   anthill = [None] * len(ants)
   return anthill
+
+def place_ant_on_anthill(anthill, ant):
+  """Insect meeple goes on correct level of home structure
+
+  Places ant onto appropriate step of the anthill structure when it travels beyond 
+  the end of the trail. This determines how many points the corresponding food tokens 
+  will be worth.
+
+  Parameters
+  ----------
+  anthill : (list)
+    A list with the same length as the number of ants in the game. 
+    Initialised with each element as None.
+    Elements will be changed into the IDs of the ants as they reach the anthill.
+    Each element is None or string.
+
+  ant : (string)
+    The ID of the ant which is being placed onto the anthill.
+
+  Returns
+  -------
+  anthill : (list)
+    Newly updated version of the anthill list; showing one fewer None and one more 
+    ant ID (string) in the appropriate place.
+  """
+  for i in range(len(anthill)-1, -1, -1):
+    if anthill[i] != None:
+      continue
```

Comment:
I've not used `continue` frequently but I can see why you've used it here and I think it's appropriate. In case you weren't aware, using `pass` in this case would achieve the same result because there is nothing else inside the for loop after the if/else. See this link for the difference between pass and continue: https://stackoverflow.com/questions/9483979/is-there-a-difference-between-continue-and-pass-in-a-for-loop-in-python/36952166